client/src/withRoot.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { MuiThemeProvider, createMuiTheme } from 'material-ui/styles';
+import purple from 'material-ui/colors/purple';
+import green from 'material-ui/colors/green';
+import CssBaseline from 'material-ui/CssBaseline';
+
+// A theme with custom primary and secondary color.
+// It's optional.
+const theme = createMuiTheme({
+    palette: {
```

Comment:
Just curious how did you choose the initial colors? Are these from the branding samples on #5 ?